### DISCUSSION TITLE
Strapi Identifier or password invalid: wrong url

### DISCUSSION THREAD

**Discussion by GinFoB** [strapi/strapi#6748] [Legacy Topics]
I have an author role, i want user with an author role connect to Strapi app for add content for example.

I found http://localhost:1337/admin/auth/login for connect to admin panel of Strapi but this for Administrator.

What's url for user with author role ?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
